demos/index.html

```diff
@@ -25,9 +28,37 @@
     </head>
 
     <body>
+
+        <div id="floec-overviewPanel" class="fl-overviewPanel-container"></div>
+
         <div id="floec-chartAuthoring"></div>
 
         <script type="text/javascript">
+            fluid.overviewPanel("#floec-overviewPanel", {
+                resources: {
+                    template: {
+                        href: "../src/lib/infusion/src/components/overviewPanel/html/overviewPanelTemplate.html"
+                    }
+                },
+                strings: {
+                    componentName: "Chart Authoring Tool",
+                    feedbackText: "Have a comment?",
+                    feedbackLinkText: "Email us"
```

Comment:
Also it's still has the "An Infusion component demo". I'm not sure that's the header we want. Although I'm not sure what to change it to in this particular case. Maybe a conversation in the channel could help sort that out.